### PR TITLE
checkpoint-sqlite: new library for sqlite checkpointer implementation

### PR DESCRIPTION
**Pull request by vbarda**

### CODE REVIEW

File: libs/checkpoint-sqlite/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    "node": {
+      "start:node": 2
+    }
+  },
+  "pending_sends": [],
+  "current_tasks": {}
+}
+
+# store checkpoint
+await checkpointer.put(thread_config, checkpoint, {})
```
Comment (nfcampos) [inline libs/checkpoint-sqlite/README.md:84]: wrong method?

Comment (vbarda) [reply libs/checkpoint-sqlite/README.md:84]: thanks!

File: libs/checkpoint-sqlite/pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+description = "Library with a SQLite implementation of LangGraph checkpoint saver."
+authors = []
+license = "MIT"
+readme = "README.md"
+repository = "https://www.github.com/langchain-ai/langgraph"
+packages = [{ include = "langgraph" }]
+
+[tool.poetry.dependencies]
+python = "^3.9.0,<4.0"
+aiosqlite = "^0.20.0"
```
Comment (nfcampos) [inline libs/checkpoint-sqlite/pyproject.toml:13]: lets update

Comment (vbarda) [reply libs/checkpoint-sqlite/pyproject.toml:13]: i think it's the latest https://pypi.org/project/aiosqlite/

Comment (nfcampos) [reply libs/checkpoint-sqlite/pyproject.toml:13]: ah cool

**Pull request merged**